### PR TITLE
Add bidsschematools

### PR DESCRIPTION
**Pull request by jwodder**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

@yarikoptic @TheChymera Please confirm willingness to be listed as maintainers.

### CODE REVIEW

File: recipes/bidsschematools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Python tools for working with the BIDS schema
+  dev_url: https://github.com/bids-standard/bids-specification
+
+extra:
+  recipe-maintainers:
+    - jwodder
+    - yarikoptic
```
Comment (yarikoptic) [inline recipes/bidsschematools/meta.yaml:56]: I confirm!

**Comment by TheChymera** [comment]
@jwodder I also confirm.

File: recipes/bidsschematools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - TheChymera
```
Comment (effigies) [inline recipes/bidsschematools/meta.yaml:57]: If you don't mind...

```suggestion
    - TheChymera
    - effigies
```

Comment (yarikoptic) [reply recipes/bidsschematools/meta.yaml:57]: not at all! @jwodder please add @effigies to this honorable list ;)

File: recipes/bidsschematools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{% set name = "bidsschematools" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a091b6a16ec01be00db76209b9a4d19005589382563a6caf503aa1140317a06f
```
Comment (effigies) [inline recipes/bidsschematools/meta.yaml:10]:
```suggestion
{% set version = "0.5.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: 139e468312171952bed71ccac4abba16010a138b8a265dddcd47372fb9bed964
```

**Comment by jwodder** [comment]
@conda-forge/help-python Ready for review.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**